controlsfx/src/main/java/org/controlsfx/control/table/ColumnFilter.java

```diff
@@ -135,6 +147,23 @@ FilterPanel getFilterPanel() {
         return filterPanel;
     }
 
+    /**
+     * Display or hides the Filter Icon in the tableColumn header.
+     * This should only be called through {@link TableFilter#executeFilter()}
+     */
+    void applyFilterIcon() {
+        if (hasUnselections()) {
+            tableColumn.setGraphic(filterImageView.get());
+            if (!bumpedWidth) {
+                tableColumn.setPrefWidth(tableColumn.getWidth() + 20);
+                bumpedWidth = true;
+            }
+        }
+        else {

```

Comment:
I don't think we follow that pattern. The `else statement` should be on the same line.